profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/features/ServerApiMockTest.java

```diff
@@ -65,7 +65,7 @@ public void testGetListWithDepth() throws InterruptedException {
    
    @Test
    public void testGetListWith404() throws InterruptedException {
```

Comment:
You may remove the test for 404 fallback, as this is already tested in the core.